test/mjs/plugins/fastify-plugin.mjs

```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+import fastifyPlugin from 'fastify-plugin'
+
+function pluginMjs (f, opts, next) {
+  f.get('/mjs-fastify-plugin', (request, reply) => {
+    reply.send({ hello: 'world' })
+  })
+
+  next()
+}
+
+export default fastifyPlugin(pluginMjs)
```

Comment:
I'm not sure this is actually valid with experimental modules, while I think it is valid with `esm`.